src/_includes/install/sampledata/sample-data-clone-24.md

```diff
@@ -87,6 +87,10 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. ```bash
```

Comment:
In the `point 8` we've changed a directory to `<magento_root>`.

Please add the next for `point 12`:

Run the following command:

```bash
bin/magento setup:upgrade
```

Thank you!

Reply:
Hi @atwixfirster 
Thanks for your notes.

Everything is done at your request.
Could you please proceed with the code review?

Reply:
Almost done.

Reply:
@atwixfirster I added an empty line.
Please proceed.
Thank you!